src/radio-statuses/observable.js

```diff
@@ -0,0 +1,21 @@
+// used in jsdoc
+// eslint-disable-next-line no-unused-vars
+import { Observable } from 'rxjs';
```

Comment:
```suggestion
import { Observable, timer } from 'rxjs';
```

Trzeba zaimportować timera